guide/xml/portfile-keywords.xml

```diff
@@ -277,5 +277,24 @@
       </listitem>
     </varlistentry>
 
+    <varlistentry>
+      <term>use_xcode</term>
+
+      <listitem>
+        <para>Available in 2.6.0 and later. By default, it is assumed on
+        <code>macosx</code>that ports will not need tools from Xcode.app
+        unless (1) CommandLineTools is not available and macOS version is
```

Comment:
I don't quite get that sentence. As I see things, we automatically assume `use_xcode yes` in the following cases:

 - `![file executable /Library/Developer/CommandLineTools/usr/bin/make]`, i.e. if the CLTs aren't installed
 - `![file exists /usr/lib/libxcselect.dylib]`, i.e. an old macOS
 - `[option build.type] eq "xcode"`

Consequently, shouldn't this read:
> that ports will not need tools from Xcode.app unless (1) Command Line Tools aren't installed, (2) you are on an old version of Mac OS X that does not support the xcode-select mechanism, or (3) the port uses `build.type xcode` or is in the `xcode` PortGroup.

Reply:
Fixed